examples/chip-tool/main.cpp

```diff
@@ -175,7 +210,8 @@ void ShowUsage(const char * executable)
             "    echo device-ip-address device-port\n"
             "    off device-ip-address device-port endpoint-id\n"
             "    on device-ip-address device-port endpoint-id\n"
-            "    toggle device-ip-address device-port endpoint-id\n",
+            "    toggle device-ip-address device-port endpoint-id\n"
+            "    read-onoff device-ip-address device-port endpoint-id\n",
```

Comment:
Maybe we can make read a top level command, and have on-off be a subcommand? (Fine with followup issue)

Reply:
Not a bad idea.  Filed https://github.com/project-chip/connectedhomeip/issues/1985